src/components/Header/AppHeader.tsx

```diff
@@ -7,38 +7,93 @@ const activeStyle: React.CSSProperties = {
+            </li>
+            <li>
+              <a href="#0">Mine utlegg</a>
+            </li>
+            <li className="text-red-500">
+              <a href="#0">Logg ut</a>
+            </li>
+          </ul>
+        </button>
+      </div>
+    </div>
+  );
+};
 
-  const linkElements = routes.map(
-    (route) => (
-      <NavLink
-        key={route.route}
-        // className="mt-4 vektor-font font-semibold p-1"
-        className="header-link mobile-link"
-        to={route.route}
-        style={({ isActive }) => (isActive ? activeStyle : {})}
+const LoginButtons = () => {
+  return (
+    <div className="flex space-x-4 w-1/5 justify-start">
+      <button
+        type="button"
+        className="bg-blue-900 hover:bg-grey-800 text-white px-6 py-2 rounded-full"
       >
-        {route.route[1].toUpperCase() + route.route.substring(2).replace("-", " ")}
-      </NavLink>
-    ),
+        Jeg er ny
+      </button>
+      <button
+        type="button"
+        className="bg-blue-400 hover:bg-blue-900 text-white px-6 py-2 rounded-full"
+      >
+        Logg inn
+      </button>
+    </div>
   );
+};
+
+const AppHeader = (): JSX.Element => {
+  const [menuOpen, setMenuOpen] = useState(false);
+  const isLoggedIn = true;
+  const linkElements = routes.map((route) => (
+    <NavLink
+      key={route.route}
+      // className="mt-4 vektor-font font-semibold p-1"
+      className="header-link mobile-link"
+      to={route.route}
+      style={({ isActive }) => (isActive ? activeStyle : {})}
+    >
+      {route.route[1].toUpperCase()
+        + route.route.substring(2).replace("-", " ")}
+    </NavLink>
+  ));
 
   return (
     <nav className="AppHeader md:sticky top-0 shadow-sm z-50">
-      <div className="hidden md:flex mx-auto p-3 space-x-8 items-center justify-center lg:px-8 w-full max-w-7xl">
+      <div className="hidden md:flex mx-auto p-3 space-x-6 items-center justify-center lg:px-8 w-full max-w-7xl">
         <div className="w-1/5 justify-center flex">
           <Link to="/">
-            <img src="images/vektor-logo.svg" alt="vektorprogrammet logo" className="h-16 lg:h-20" />
+            <img
+              src="images/vektor-logo.svg"
+              alt="vektorprogrammet logo"
+              className="h-16 lg:h-20"
+            />
           </Link>
         </div>
-        <div className="flex flex-grow justify-between justify-self-center max-w-lg m-auto items-center w-2/5">
+        <div className="flex flex-grow justify-evenly justify-self-center max-w-lg m-auto items-center w-2/5">
           {linkElements}
         </div>
-        <div className="flex space-x-4 w-1/5 justify-start">
-          <button type="button" className="bg-blue-900 hover:bg-grey-800 text-white px-6 py-2 rounded-full">Jeg er ny</button>
-          <button type="button" className="bg-blue-400 hover:bg-blue-900 text-white px-6 py-2 rounded-full">Logg inn</button>
-        </div>
+        {isLoggedIn ? <LoginButtons /> : <UserAvatar />}
```

Comment:
Add a wrapping `div`, 
These should probably be other way around as well?
```suggestion
    <div className="flex justify-center w-1/5">
        {isLoggedIn ? <UserAvatar /> : <LoginButtons />}
    </div>
```